packages/components/src/UICompatibilityView/index.web.js

```diff
@@ -1,17 +0,0 @@
-// @flow
-import React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-// TODO: we have `mobile-detect` in UIDevice
-// maybe use only one lib for
-import Bowser from 'bowser';
```

Comment:
Btw, we can remove the `bowser` from dependencies then

Reply:
@sertony done!